pipeline/test/services/service-cluster/testPodsReady.sh

```diff
@@ -206,23 +206,26 @@ if "${enable_opensearch_snapshot}"; then
     )
 fi
 if "${enable_fluentd_audit}"; then
-    audit_bucket="$(yq4 '.objectStorage.buckets.audit' "${CONFIG_FILE}" 2>/dev/null)"
-    env_name="$(yq4 '.global.ck8sEnvironmentName' "${CONFIG_FILE}" 2>/dev/null)"
+    cluster_name="$(yq4 '.global.clusterName' "${CONFIG_FILE}" 2>/dev/null)"
+    #clusters_monitoring=($(yq4 -r '.global.clustersMonitoring[]' "${CONFIG_FILE}" 2>/dev/null))
+    mapfile -t clusters_monitoring < <(yq4 -r '.global.clustersMonitoring[]' "${CONFIG_FILE}" 2>/dev/null)
```

Comment:
Is `-r` a valid flag of `yq`?

Reply:
yes, but I will remove it

>   -r, --unwrapScalar                  unwrap scalar, print the value with no quotes, colors or comments (default true)

Reply:
I see, I managed to be outdated by 1 minor version :sweat_smile:

Reply:
removed as it is true by default